dynamic/parameterize/args.go

```diff
@@ -72,14 +74,27 @@ func ParseArgs(ctx context.Context, a []string) (Args, error) {
 	var indexDocOutDir string
 	var includes []string
 	var providerName string
+
+	// If --help is included in `a`, then `RunE` will never be executed.
```

Comment:
nit because I stumbled over this a bit: I see `a` on line 85. Can we rename it to "args"? 🙏